package.json

```diff
@@ -91,12 +91,13 @@
     "fix:css": "stylelint public/css/ --fix",
     "test:db:migrate": "NODE_ENV=tests knex migrate:latest --knexfile db/knexfile.js --env tests",
     "test:tests": "NODE_ENV=tests HIBP_THROTTLE_DELAY=1000 HIBP_THROTTLE_MAX_TRIES=3 jest --runInBand --coverage tests/",
+    "test:is_coveralls_configured": "node scripts/is_coveralls_configured.js",
     "test:coveralls": "cat ./coverage/lcov.info | coveralls",
     "test:integration": "wdio tests/integration/wdio.conf.js",
     "test:integration-headless": "MOZ_HEADLESS=1 wdio tests/integration/wdio.conf.js",
     "test:integration-headless-ci": "MOZ_HEADLESS=1 ERROR_SHOTS=1 wdio tests/integration/wdio.conf.js",
     "test:integration-docker": "MOZ_HEADLESS=1 wdio tests/integration/wdio.docker.js",
-    "test": "npm run test:db:migrate && npm run test:tests && npm run test:coveralls"
+    "test": "npm run test:db:migrate && npm run test:tests && (npm run test:is_coveralls_configured && npm run test:coveralls || echo 'Skipping coveralls.')"
```

Comment:
Nit: there might be less noise if we just directly call `node scripts/is_coveralls_configured.js` rather than creating a separate script for it then calling the similarly long `npm run test:is_coveralls_configured`?